js_modules/dagster-ui/packages/ui-core/src/resources/WorkspaceResourcesRoot.tsx

```diff
@@ -141,9 +141,7 @@ export const RESOURCE_ENTRY_FRAGMENT = gql`
       path
     }
     jobsOpsUsing {
-      job {
-        id
-      }
+      jobName
```

Comment:
weird that we went from id to name, were we using either?

Reply:
I think it just needed to fetch something to be able to get objects back to populate the list and then we just looked at the length of the list